src/Raven.Server/Documents/Changes/AbstractChangesClientConnection.cs

```diff
@@ -191,7 +191,7 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ms))
                                     context.Write(writer, djv);
                                     break;
                                 case BlittableJsonReaderObject bjro:
-                                    context.Write(writer, bjro);
+                                    context.Write(writer, bjro.Clone(context));
```

Comment:
why?

Reply:
changed to `CloneForConcurrentRead` so different context will be used for reading